plugins/actuator/FMIActuatorPlugin.cc

```diff
@@ -209,8 +214,14 @@ bool FMIActuatorPlugin::ParseParameters(gazebo::physics::ModelPtr _parent, sdf::
         gzwarn << "FMIActuatorPlugin: option jointTorqueName deprecated, please use variable_names tag instead." << std::endl;
         actuator.m_outputVariablesNames[FMIActuatorPluginNS::jointTorque] = elem->Get<std::string>("jointTorqueName");
       }
-
+    
+      if(verbose)
+      {
+        gzwarn << "FMIActuatorPlugin: actuator loaded for joint name:" << jointName << std::endl;
```

Comment:
Perhaps `gzinfo` is more appropriate? If the plugin is loaded, I do not think we should consider it a warning. 
```suggestion
        gzinfo << "FMIActuatorPlugin: actuator loaded for joint name:" << jointName << std::endl;
```

Reply:
Ok but I was not able to see on the log if I use gzinfo. Is it due to gzsever log severity?